.github/workflows/windturbine-sampleapp-pull-request.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Apache v2 license
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+name: "[Time Series] PR workflow"
+run-name: "[Time Series] PR  workflow (by @${{ github.actor }} via ${{ github.event_name }})"
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'manufacturing-ai-suite/wind-turbine-anomaly-detection/*'

```

Comment:
This is more for push, we shouldn't be having this as we don't care for pushes happening to `main` branch and care only for PR branches.
Mainly we need to have this for pull_request https://github.com/intel-innersource/frameworks.ai.ai-suite-for-timeseries/blob/main/.github/workflows/pull_request.yml#L7-L15.